src/accordion/accordion.component.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { TestBed } from "@angular/core/testing";
+import { FormsModule } from "@angular/forms";
+import { By } from "@angular/platform-browser";
+import { Component } from "@angular/core";
+import { ChevronRight16Module } from "@carbon/icons-angular/lib/chevron--right/16";
+import { AccordionItem } from "./accordion-item.component";
+import { Accordion } from "./accordion.component";
+
+@Component({
+	template: `
+	<ibm-accordion>
+		<ibm-accordion-item
+		(selected)="onClick()"
+		title="Section 1"
+		[skeleton]="false">
+		test-content
+	</ibm-accordion-item>`
+})
+class AccordionTest {
+	onClick() {}
+}
+
+describe("Accordion", () => {
+	let fixture, wrapper, element;
+	beforeEach(() => {
+		TestBed.configureTestingModule({
+			declarations: [
+				Accordion,
+				AccordionItem,
+				AccordionTest
+			],
+
```

Comment:
```suggestion
```

Reply:
It does not run when I remove these two declarations.

Reply:
If you commit this suggestion, you'll notice that it's only removing the empty line :)

Reply:
Oh I see :)